editor/state.js

```diff
@@ -34,7 +40,20 @@ export const blocks = combineUndoableReducers( {
 
 		return state;
 	},
-	byUid( state = {}, action ) {
+
+	edits( state = {}, action ) {
+		switch ( action.type ) {
+			case 'EDIT_POST':
```

Comment:
The idea is that we'll use it in follow-up pull requests, definitely for non-content properties like status, author, taxonomies, ...

Still unclear to me: whether/when to insert the post content into the `edits` state key.  For example, I can imagine something like the Calypso editor where the content is requested from the editor 200ms after the user stops typing.  This seems like potentially a lot of overhead from frequent content serialization though.